blocks/map/map.js

```diff
@@ -1,8 +1,22 @@
 import {
-  div, iframe, p, horizontalRule,
+  div, iframe, p, horizontalRule, domEl,
 } from '../../scripts/dom-helpers.js';
 
-function getMapFrame(url) {
+const getMapPlaceholder = function () {
```

Comment:
Maybe you could change this into an arrow function, since github is also giving a warning about this
```suggestion
const getMapPlaceholder = () => {
```